api.js

```diff
@@ -1,9 +1,17 @@
 var mongo = require('mongodb').MongoClient,
-    fiddles = null;
+    fiddles = null,
+    mongoConnectionStr = process.env.MONGOHQ_URL ?
+        String(process.env.MONGOHQ_URL) :
+        'mongodb://localhost:27017/es6-fiddle';
 
 module.exports = function(app) {
-    mongo.connect(String(process.env.MONGOHQ_URL), function(err, db) {
-        fiddles = db.collection('fiddles');
+    
+    mongo.connect(mongoConnectionStr, function(err, db) {
+        if (err) {
+            console.error('✗ MongoDB Connection Error ', err);
```

Comment:
I would probably not put unicode in this string (that x character) in case someone is using an ACSI terminal.